build.sbt

```diff
@@ -27,27 +24,30 @@ version in Docker := "1.0"
 organization := "com.codacy"
 
 val installAll =
-  s"""echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories &&
-     |apk --no-cache add bash wget ca-certificates git &&
-     |apk --no-cache add python &&
-     |apk --no-cache add 'python3>3.6.1' &&
-     |wget "https://bootstrap.pypa.io/get-pip.py" -O /dev/stdout | python &&
-     |wget "https://bootstrap.pypa.io/get-pip.py" -O /dev/stdout | python3 &&
-     |python -m pip install django==1.9.2 pylint-django==0.9.0 flask==0.10.1 pylint-flask==0.1 flask-wtf==0.12 --upgrade --ignore-installed --no-cache-dir &&
-     |python3 -m pip install django==1.9.2 pylint-django==0.9.0 flask==0.10.1  pylint-flask==0.1 flask-wtf==0.12 --upgrade --ignore-installed --no-cache-dir &&
-     |python -m pip install pylint-common==0.2.2 &&
-     |python3 -m pip install pylint-common==0.2.2 &&
-     |python -m pip install pylint-celery==0.3 &&
-     |python3 -m pip install pylint-celery==0.3 &&
-     |python -m pip install SaltPyLint==2017.12.15 &&
-     |python3 -m pip install SaltPyLint==2017.12.15 &&
-     |python -m pip install pylint==1.8.2 --upgrade --ignore-installed --no-cache-dir &&
-     |python3 -m pip install pylint==1.8.2 --upgrade --ignore-installed --no-cache-dir &&
-     |python -m pip uninstall -y pip &&
-     |python3 -m pip uninstall -y pip &&
-     |apk del wget ca-certificates git &&
-     |rm -rf /tmp/* &&
-     |rm -rf /var/cache/apk/*""".stripMargin.replaceAll(System.lineSeparator()," ")
+  s"""apt-get update &&
+      |apt-get -y install python python3 ca-certificates wget openjdk-8-jre-headless &&
```

Comment:
won't this install the most recent version for python and python3? I think we should install always the same python version for the build. 

`sudo apt-get install python2.7`
`sudo apt-get install python3.6`

Does this work?